src/AuthLDAP.php

```diff
@@ -233,6 +235,8 @@ public function preconfig($type)
                 $this->fields['pagesize']                  = '1000';
                 $this->fields['picture_field']             = '';
                 $this->fields['responsible_field']         = 'manager';
+                $this->fields['begin_date_field']          = 'whenCreated';
```

Comment:
Maybe it should not be part of preconfig if these fields are available only on Microsoft ldaps.

Reply:
These default values are only set for AD (see condition above) so they should be available in this case.
That being said, simply being available does not mean it must be used as the default value.

I think the second one make sense (accountExpires) as it represent a real end date in the AD that will impact GLPI once it's reached so we could keep it as a default value.

The first one (whenCreated) represent the date at which the user was created in the AD.
It has no impact on GLPI as the user can't be imported before it was created anyway (so any imported user will always have whenCreated > current date).
I guess it does give the information of the AD creation date inside GLPI so it may not be entirely useless, but is it enough to justify using it as the default value ?

Reply:
I did not see we were in `AD` case.
I guess that it make sense to use these information as long as they are available.